packages/api/src/auth.js

```diff
@@ -1,6 +1,7 @@
 import * as JWT from './utils/jwt.js'
 import {
   UserNotFoundError,
+  PinningNotAuthorisedError,
```

Comment:
Perhaps more commonly?:

```suggestion
  PinningUnauthorizedError,
```

Reply:
Ok cool, spelling changed throughout.